.circleci/config.yml

```diff
@@ -124,7 +124,7 @@ jobs:
       - image: circleci/ruby:2.4
     working_directory: /tmp/deploy
     environment:
-      - DISTROS: "trusty xenial bionic el6 el7"
+      - DISTROS: "trusty xenial bionic el6 el7 el8"
```

Comment:
You need to change `parallelism` as well in the same `circleci` config.
Otherwise your `el8` job is not running, according to CI build which has 5 jobs.

Reply:
done